### PR TITLE
Update usingr

### PR DESCRIPTION
**Pull request by fawda123**
Just finished porting over content for the first module.  Let me know what you think - is it too long for an hour?

### CODE REVIEW

File: usingr.Rmd [hunk 1]
```diff
@@ -0,0 +1,359 @@
...
+
+# Using R for bioassessment data: What, why, and how
+
+## Lesson Outline
+
+- [Goals and Motivation]
+- [RStudio]
+- [R language fundamentals]
+- [Data structures in R]
+- [Data I/O]
```
Comment (ryanpeek) [inline usingr.Rmd]: I would actually say something like "Getting Data In/Out of R". Some folks won't know about I/O.

File: usingr.Rmd [hunk 2]
```diff
@@ -0,0 +1,359 @@
...
+![](figure/rstudio.png)
+
+There are four panes in RStudio: 
+
+* Source
+* console
+* Environment, History, etc.
+* Files, plots, etc.
+
+### RStudio projects
```
Comment (ryanpeek) [inline usingr.Rmd:65]: This is good, will you talk about folder structures/management within this section? Should we explicitly add some content/challenge asking folks to create "data", "scripts" folders and create/name a script "learning_r_at_CABW.R".

File: usingr.Rmd [hunk 3]
```diff
@@ -0,0 +1,359 @@
...
+library("tidyverse")
+library("sf")
+library("mapview")
+    ```
+
+### Getting Help
+
+Being able to find help and interpret that help is probably one of the most important skills for learning a new language.  R is no different. Help on functions and packages can be accessed directly from R, can be found on CRAN and other official R resources, searched on Google, found on StackOverflow, or from any number of fantastic online resources. I will cover a few of these here. 
+
+### Help from the console
```
Comment (ryanpeek) [inline usingr.Rmd:192]: Maybe add a link or mention the difference between "warnings" and "errors" and what each means (or how to use that information to google stuff).

**Review by ryanpeek — APPROVED** [review]
Nice work. Overall I think this is good, but may be a bit long. Some of the text you could skim over a bit more than others, or just work through certain parts more quickly. Up to you. I added a few comments, take 'em or leave 'em.

File: usingr.Rmd (hunk 2 above)
Comment (fawda123) [reply usingr.Rmd:65]: Yep, this is covered a bit in exercise one.  I also made a note to explicitly call out data/file strucure in exercise 3.

File: usingr.Rmd (hunk 1 above)
Comment (fawda123) [reply usingr.Rmd]: Ah, good call.  I changed this to "Getting your data into R".

File: usingr.Rmd (hunk 3 above)
Comment (fawda123) [reply usingr.Rmd:192]: Good call, added 29d7eb968d6869b49c484e028c6ed4bc911b8c01

**Pull request merged**